### ISSUE TITLE
Setup report for printing marathon maps

### ISSUE THREAD

**Issue by NyakudyaA** [kartoza/maps.kartoza.com#18]
# Problem

Chris currently generates maps for each segment manually and it takes him a long time to print all the maps.

# Solution

Show him how to use the reporting functionality in QGIS to automate the printing of maps

**Comment by gubuntu** [comment]
yes great, we had talked about that (won't atlas do?) - just check that all support hours stay within the contract else we might need a top-up
